go.mod

```diff
@@ -10,4 +10,10 @@ require (
 	github.com/pkg/browser v0.0.0-20210911075715-681adbf594b8
 )
 
-require golang.org/x/sys v0.5.0 // indirect
+require (
```

Comment:
Are all these ok to use as per the Open Source guidelines? https://repos.opensource.microsoft.com/